Makefile

```diff
@@ -219,10 +223,10 @@ clean_uta:
           $(OUT_DIR)/$(OUTLIB)* $(OUT_DIR)/util/$(OUTBIN) $(OUT_DIR)/util/icvutil.o
 
 clean:
+	$(MAKE) -C util clean
```

Comment:
This is inconsistent - `$(MAKE) util` is still part of `build_all`,
but you moved `$(MAKE) -C util clean` from `clean_all` to `clean`.